package.json

```diff
@@ -68,6 +67,7 @@
     "jest": "^24.9.0",
     "prettier": "^1.16.4",
     "pretty-format": "^25.2.6",
+    "request": "^2.88.0",
```

Comment:
Do we need to actually keep this? I'm fine with taking it out completely, not even having it as a dev dependency.  Same with line 60, `"@types/request": "^2.48.4"`, we can probably take both lines out!

Reply:
I still see it in tests

Reply:
Ah, I see. Unused in tests :)

Reply:
Removed